ang/crmMosaico/TemplateItem.js

```diff
@@ -12,13 +13,26 @@
       },
       link: function($scope, $el, $attr) {
         var ts = $scope.ts = CRM.ts('mosaico');
-        $scope.$watch('crmMosaicoTemplateItem', function(newValue){
+
+        $scope.$watch('crmMosaicoTemplateItem', function(newValue) {
           $scope.myOptions = newValue;
+
+          // Template default action label based on current state
+          if (newValue['state']) {
+            var clickActionLabels = {
+              select: "Select",
+              selected: "Edit",
+              configure: "Edit",
+              new: "New"
+            };
+            var clickActionLabel = clickActionLabels[newValue['state']];
+            $scope.actionLabel = clickActionLabel;
```

Comment:
Yea the problem is that this label action is different from state to another. so it could be "Edit", "Select" or maybe "New"

so i can't give it a specific name, but i can make it something like mainClickActionLabel